cognee/tests/test_neptune_analytics_graph.py

```diff
@@ -223,5 +225,77 @@ async def main():
     else:
         print(f"Delete failed")
 
+
+async def misc_methods():
+    print("------TRUNCATE GRAPH-------")
+    await na_adapter.delete_graph()
+
+    print("------SETUP TEST ENV-------")
+    nodes, edges = setup()
+    await na_adapter.add_nodes(nodes)
+    await na_adapter.add_edges(edges)
+
+    print("------GET GRAPH-------")
+    all_nodes, all_edges = await na_adapter.get_graph_data()
+    print(f"found {len(all_nodes)} nodes and found {len(all_edges)} edges")
+
+    print("------GET DISCONNECTED-------")
+    nodes_disconnected = await na_adapter.get_disconnected_nodes()
+    print(nodes_disconnected)
+    assert len(nodes_disconnected) == 0
+
+    print("------GET PREDECESSORS-------")
+    nodes_predecessors = await na_adapter.get_predecessors(
+        node_id="dummy_node_id", edge_label="test_label"
+    )
+    print(nodes_predecessors)
+    assert len(nodes_predecessors) == 0
+
+    print("------GET SUCCESSORS-------")
+    nodes_successors = await na_adapter.get_successors(
+        node_id="dummy_node_id", edge_label="test_label"
+    )
+    print(nodes_successors)
+    assert len(nodes_successors) == 0
+
+    print("------Remove connection (Predecessors)-------")
+    await na_adapter.remove_connection_to_predecessors_of(
```

Comment:
consider this test: 
```
nodes_predecessors  = await na_adapter.get_predecessors(node_id, edge_label)
asset len(nodes_predecessors) > 0

await na_adapter.remove_connection_to_predecessors_of([node_id], edge_label)

nodes_predecessors  = await na_adapter.get_predecessors(node_id, edge_label)
asset len(nodes_predecessors) == 0
```
can we do this?  That way we know the two functions are doing something.

Reply:
you could throw in a `get_relationship_labels_string` test too?